components/styled/table/src/index.tsx

```diff
@@ -1,7 +1,22 @@
 import { FC } from 'react'
 import { Questionnaire, QuestionnaireItem, QuestionnaireResponse, SummaryTableViewType } from '@ltht-react/types'
-import { EnsureMaybeArray } from '@ltht-react/utils'
+import { EnsureMaybe, EnsureMaybeArray } from '@ltht-react/utils'
 import QuestionnaireTable from './molecules/questionnaire-table'
+import GenericTableMolecule from './molecules/generic-table'
+import { TableData } from './atoms/table'
+
+interface IProps {
```

Comment:
Again I know it's a super minor point so sorry to be annoying, but could we just move these interfaces down to the bottom?